scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -4,58 +4,96 @@ class_name PlayerMovementComponent extends BaseMovementComponent
 ## The max horizontal speed the entity can move.
 @export var MAX_SPEED: int = 200
 ## The jump velocity of the entity.
-@export var JUMP_VELOCITY: int = -300
+@export var JUMP_VELOCITY: int = -350
 ## The horizontal movement acceleration for the entity.
 @export var ACCELERATION: int = 1500
 ## The time it takes to fully decelerate from horizontal input in seconds.
 @export_range(0, 10, 0.1, "or_greater") var DECELERATION_TIME: float = 0.2
 
+## Moth Jump variables
+@export var MOTH_JUMP_HANGTIME: float = 1.0
+@export var MOTH_JUMP_DIP_AMOUNT: float = -18.0
```

Comment:
Also should just be positive and adjusted in actual code sections

Reply:
Used _ready to invert